regression_and_census.R

```diff
@@ -9,9 +9,166 @@
 # old code-  I M P O R T I N G   C E N S U S   D A T A ---------------------------------------
 
 library(latticeExtra)
-install.packages("gee")
+library(gee)
+library(geepack)
+library(MASS)
+library(broom)
 # tidyverse
 library(dplyr)
+library(readr)
```

Comment:
In general it's a good idea to load all the libraries you need at the top of each script so the code in each source file can run independently.